### PR TITLE
[CBRD-26572] Add supplemental test cases for UUID function support

### PR DESCRIPTION
**Pull request by kimhak-tech**
Refer to http://jira.cubrid.org/browse/CBRD-26572.

This PR supplements the existing test cases added in CUBRID/cubrid-testcases#2810.

### Specification

Based on the CBRD-26572 requirements and the implementation discussion in CUBRID/cubrid#7213:

- The argument of `UUID()` is coerced to `INTEGER` before UUID version dispatch.
- Decimal arguments are rounded during this coercion.
- For example:
  - `UUID(4.111)` is rounded to `UUID(4)` and generates UUIDv4.
  - `UUID(7.321)` is rounded to `UUID(7)` and generates UUIDv7.
  - Decimal values that round to unsupported versions must raise an error.

### Implementation

- Add test cases for decimal arguments passed to `UUID()`.
- Cover supported rounded values, unsupported rounded values, and boundary values around UUIDv4 and UUIDv7.
- Verify the same behavior for:
  - literal arguments
  - string-form numeric arguments
  - `DEFAULT UUID(...)` expressions
  - column-sourced DML arguments
- Add one supplemental `UUID_FORMAT()` boundary case for bit input longer than 128 bits.

### CODE REVIEW

File: sql/_36_guava/cbrd_26486/cases/01_uuid_basic.sql [hunk 1]
```diff
@@ -100,3 +100,41 @@ from (select cast(conv(concat(substr(uuid_format(u), 1, 8), substr(uuid_format(u
...
+
+evaluate '[TEST 16] rounding boundary values that resolve to supported UUID versions';
+-- Values below 0.5 round to 0, which is accepted as UUIDv4.
+select substr(uuid_format(uuid(0.00)), 15, 1) v_0_00,
+       substr(uuid_format(uuid(0.49)), 15, 1) v_0_49;
+-- 3.5 and higher values below 4.5 round to 4, which is accepted as UUIDv4.
+select substr(uuid_format(uuid(3.5)), 15, 1) v_3_5,
+       substr(uuid_format(uuid(3.9)), 15, 1) v_3_9,
+       substr(uuid_format(uuid(4.49)), 15, 1) v_4_49;
+-- 6.5 and higher values below 7.5 round to 7, which is accepted as UUIDv7.
```
[the remark this answers is not in the document]
Comment (kimhak-tech) [reply sql/_36_guava/cbrd_26486/cases/01_uuid_basic.sql:136]: The expected result is correct.

`UUID()` coerces its argument to `INTEGER` before selecting the UUID version, and in the current implementation `6.5` is rounded to `7`. Therefore `uuid(6.5)` is expected to generate UUIDv7, not raise an unsupported-version error.

**Comment by bagus-kim** [comment]
TC가 체계적으로 잘 작성되어 있네요. answer 파일 대응, 에러 코드, cleanup, 번호 연속성 모두 문제 없습니다.

[선택 사항] uuid(-0.5) 경계값 (양수 .5 경계와의 대칭성) - 값을 보다가 -0.5가 있으면 좋지 않을까 하는 생각이 들어서 작성한 것이니 담당자분이 선택하시면 되겠습니다.

양수 쪽 .5 경계는 모두 명시적으로 테스트되어 있습니다 (0.5, 3.5, 4.5, 6.5, 7.5). 그런데 음수 쪽은 -0.51 (에러)과 -0.49 (성공)만 있고, 정확히 -0.5는 테스트되지 않습니다.

**Review by bagus-kim — APPROVED** [review]

**Review by childyouth — APPROVED** [review]

**Review by ssihil — APPROVED** [review]

**Review by kwonhoil — APPROVED** [review]

**Review by junsklee — APPROVED** [review]

**Review by kiho-um — APPROVED** [review]
